helpers/theme.js

```diff
@@ -29,11 +29,23 @@ const instanceInstitute = () => {
 	}
 };
 
+const instanceCloudConsentNecessary = () => {
```

Comment:
I think the switch case is not necessary here as it confuses the reading flow too much. As I understood there are only some cases needed where its false and the rest is true. so a simplified if condition would suit better I think to make the white-flagging more visible.

Reply:
I agree that the if statement makes it easier to read. Changed :)